src/test/ERC1155.t.sol

```diff
@@ -14,10 +14,116 @@ contract ERC1155Test is DSTestPlus {
         token = new MockERC1155();
     }
 
-    function testSupportsERC165Interface() public {
-        // assertTrue(token.supportsInterface(0x01ffc9a7));
-        // assertTrue(token.supportsInterface(0xd9b67a26));
+    function testMint() public {
+        token.mint(address(0xBEEF), 1, 1, "");
+
+        assertEq(token.balanceOf(address(0xBEEF), 1), 1);
+    }
+
+    function testBatchMint(
+        address usr,
+        uint256[] memory ids,
+        uint amount
+    ) public {
+        if (usr == address(0)) return;
+        
+        uint len = ids.length;
+        uint256[] memory amounts = new uint256[](len);
+        address[] memory owners = new address[](len);
+
+        for (uint i = 0; i < len; i++) {
+            //if ids array has any dupes amounts will not be correct
+            for (uint256 j = 0; j < i; j++) {
+                if (ids[i] == ids[j]) return;
+            }
```

Comment:
These shit for fuzzing times but will fail with dupes otherwise. Could use a better test solution or just ignore the fails